Automattic/delphin
# Update contact forms to send HEs an email

Fixes #714.

This PR updates `ContactUsExistingBlog` to send an email to HEs when submitted.

**Testing**
- Visit `MyDomains`.
- Click `Set Up Now`.

_New blog_
- Click `A blog I've already created`.
- Enter the address of a non-WordPress.com site (e.g. `youtube.com`) and submit.
- Enter a message in this form and submit.
- Assert that you are redirected back to `MyDomains` with a success notice.

_Existing blog_
- Click `A blog I am going to create`.
- Click `Somewhere else`.
- Enter a message in this form and submit.
- Assert that you are redirected back to `MyDomains` with a success notice.

cc @spncrb 
#### Reviews
